documentation/docs/identus/cloud-agent/overview.md

```diff
@@ -43,39 +43,39 @@ This document provides an overview of the Cloud Agent feature set. This document
 
 | Credential Type | Supported | Notes |
 | --- | :--: | -- |
-| [W3C Standard Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) | :white_check_mark: | Supports JSON and JSON-LD Credentials encoded as JWT using the `JsonWebSignature2020` with `ESK256K` signature suite.<br /><br />Supports the [DIF Presentation Exchange](https://identity.foundation/presentation-exchange/) data format for presentation requests and presentation submissions. |
+| [W3C Standard Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) | :white_check_mark: | Supports JSON and JSON-LD Credentials encoded as JWT using the `JsonWebSignature2020` with `ES256K` (secp256k1) signature suite.<br /><br />Supports the [DIF Presentation Exchange](https://identity.foundation/presentation-exchange/) data format for presentation requests and presentation submissions. |
```

Comment:
@elribonazo can you review this. I have no idea what `ESK256K` is I believe it's a typo

Reply:
ES256K, correct. Typo